index.html

```diff
@@ -6,36 +6,26 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="./assets/css/styles.css">
-    <title>First Project</title>
+    <title>Javier Martinez's Portfolio</title>
 </head>
 
 <body>
     <header>
-        <div class="topHeader">
-            <div class="logo">
-                <a href="#" rel="noopener" target="_blank">
-                    My Logo
-                </a>
-            </div>
-            <nav>
-                <ul class="links-index">
-                    <li>
-                        <a href="#">Portfolio</a>
-                    </li>
-                    <li>
-                        <a href="#">About</a>
-                    </li>
-                    <li>
-                        <a href="#">Contact</a>
-                    </li>
-                </ul>
-                <div id="breadcrumb">
-                    <a href="#" rel="noopener" target="_blank">
-                        <img src="./assets/images/breadcrumb.png" alt="">
-                    </a>
-                </div>
-            </nav>
+        <div class="logo">
+          <a href="#">My Logo</a>
         </div>
+  
+        <nav>
+          <div class="hamburger">
+            <img src="./assets/images/breadcrumb.png" alt="" />
+          </div>
+          <div class="main-menu">
+            <span class="close">X</span>
+              <a class="links" id="Portfolio" href="#portfolio">Portfolio</a>
+              <a class="links" id="about" href="#about">About</a>
```

Comment:
- [ ] As it is above, clicking on the About is not traversing to the About section of the page.

Reply:
[observation1.webm](https://user-images.githubusercontent.com/86327867/231969831-5996d553-1afa-44fc-b7c0-95603395a6be.webm)

As you can see in the video, I resolved the issue about the sections.

Reply:
![03](https://user-images.githubusercontent.com/86327867/231979048-ebd8cb8e-fff2-4dd1-aae4-c16f9a23131b.jpg)

Switching to About Myself section.